scripts/genetic_algo/parsing.py

```diff
@@ -0,0 +1,25 @@
+import shlex
+import subprocess
+
+
+def parse_trades(stuff):
+    """
+    >>> parse_trades("1 trades over 17 days (avg 0.06 trades/day)")
+    '0.06'
+    :param stuff:
+    :return:
+    """
+    return stuff.split(b'avg')[-1].strip().split()[0]
+
+
+def args_for_strategy(strat):
+    available = subprocess.check_output(shlex.split('/app/zenbot.sh list-strategies'))
+    strats = [strat.strip() for strat in available.split(b'\n\n')]
```

Comment:
`import re`

`ansi_escape = re.compile(b'\x1b[^m]*m')`
`available = subprocess.check_output(shlex.split('/app/zenbot.sh list-strategies'))`
`strats = [ansi_escape.sub(b'', strat.strip()) for strat in available.split(b'\n\n')]`

Could I propose these lines to get rid of ANSI escape sequences? otherwise it seems to me that this function is not able to parse correctly command options. NB: I'm using python scripts outside docker container.

Reply:
Added.